### PR TITLE
Add ar1 lecture

### PR DESCRIPTION
**Pull request by jstac**

### CODE REVIEW

File: source/rst/ar1_processes.rst [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+
+.. math::
+
+    M_k := \mathbb E [ (X - \mathbb E X )^k ]
+
+When that random variable is :math:`N(\mu, \sigma^2)`, it is know that
+
+.. math::
+
+    M_k =
```
Comment (RaymondMcT) [inline source/rst/ar1_processes.rst]: The cases should be flipped
```M_k = 
    \begin{cases}
        0 & \text{ if } k \text{ is odd} \\
        \sigma^k (k-1)!! & \text{ if } k \text{ is even} 
    \end{cases}```

Comment (jstac) [reply source/rst/ar1_processes.rst]: I think it's correct, isn't it?  For example, consider k=1 or k=2.

Comment (RaymondMcT) [reply source/rst/ar1_processes.rst]: In your solution you have k=6 (even) ~ 35, yet you say it should be zero.

k=1 is just E[X] - E[X] = 0
k=2 E[X - E[X]]^2 = Var(x) != 0

Sorry I corrected the code in my comment, didn't just quote.

Comment (jstac) [reply source/rst/ar1_processes.rst]: Ah, sorry, now I'm with you.  Thanks for picking up my silly error `:-)`.  I'll fix it.

I greatly appreciate your input @RaymondMcT.  The lecture was written in a bit of a rush (see #817) so there might be other typos.

Comment (jstac) [reply source/rst/ar1_processes.rst]: Fixed in https://github.com/QuantEcon/lecture-source-py/commit/3d14d3e372bdc17cc65e192551363149aa7e68d1

**Pull request merged**